src/main/java/com/example/cabbooking/controller/CabBookingController.java

```diff
@@ -4,8 +4,9 @@
 import com.example.cabbooking.servicetest.RideService;
 import com.example.cabbooking.servicetest.UserService;
 import com.example.cabbooking.vo.DriverDetailsVO;
-import com.example.cabbooking.vo.LocationDetailsVO;
 import com.example.cabbooking.vo.RideDetailsVO;
+import com.example.cabbooking.vo.LocationDetailsVO;
+
 import com.example.cabbooking.vo.UserDetailsInVO;
 import lombok.RequiredArgsConstructor;
 import org.slf4j.Logger;
```

Comment:
always try to use deep import line no 17

Reply:
yes will follow this now